tests/wpt/mozilla/tests/mozilla/bluetooth/getCharacteristics/correct-characteristics.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/bluetooth/bluetooth-helpers.js"></script>
+<script>
+'use strict';
+promise_test(() => {
+    window.testRunner.setBluetoothMockDataSet(adapter_type.heart_rate);
+    return window.navigator.bluetooth.requestDevice({
+        filters: [{services: [heart_rate.name]}]
+    })
+    .then(device => device.gatt.connect())
+    .then(gattServer => gattServer.getPrimaryService(heart_rate.name))
+    .then(service => service.getCharacteristics(body_sensor_location.name))
+    .then(characteristics => {
+        for (let i = 1; i < characteristics.length; i++) {
+            characteristics[i].readValue()
```

Comment:
We probably need to store each promise in a vector, and return Promise.all(promises) from this function.